CMakeLists.txt

```diff
@@ -37,7 +37,6 @@ add_install_rpath_support(BIN_DIRS "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BIND
 
 
 add_subdirectory(librFSM)
-add_subdirectory(examples)
+#add_subdirectory(examples)
```

Comment:
Why did you commented out the examples directory?

Reply:
I commented out because of https://github.com/robotology/rfsmTools/pull/15#discussion_r249023272. But in bd0b344 I restored the examples folder in the building phase by following the hint of @traversaro https://github.com/robotology/rfsmTools/pull/15#discussion_r249037391